DnDTests/Converters/CoinConverterTest.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Converters;
+using DTOs;
+using Enums;
+using Interfaces;
+using Models;
+
+namespace DnDTests.Converters
+{
+    internal class CoinConverterTest
+    {
+        private IConverter<Coin, CoinDTO> _mapper;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _mapper = new CoinConverter();
+        }
+        [Test]
+        public void Convert_Capability_ReturnCoinDTO()
+        {
+            // Arrange
+            Coin coin = new()
+            {
+                Id = 1,
+                Name = "Test",
+                Initials = "TT",
+            };
+
+            CoinDTO expected = new()
+            {
+                Id = 1,
+                Name = "Test",
+                Initials = "TT",
+            };
+
+            // Act
+            CoinDTO actual = _mapper.Convert(coin);
+
+            // Assert
+            Assert.Multiple(() =>
+            {
+                Assert.That(actual.Id, Is.EqualTo(expected.Id));
+                Assert.That(actual.Name, Is.EqualTo(expected.Name));
+                Assert.That(actual.Initials, Is.EqualTo(expected.Initials));
+            });
+        }
```

Comment:
use var to allow compiler to infer types 
https://learn.microsoft.com/es-es/dotnet/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables